sdk/springappdiscovery/Azure.ResourceManager.SpringAppDiscovery/Azure.ResourceManager.SpringAppDiscovery.sln

```diff
@@ -0,0 +1,79 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30309.148
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{D5740887-B929-4A8E-A2B7-6113F059F929}") = "Azure.ResourceManager.SpringAppDiscovery", "src\Azure.ResourceManager.SpringAppDiscovery.csproj", "{500DF848-0E04-498F-9E1D-AB724AB99D20}"
+EndProject
+Project("{D5740887-B929-4A8E-A2B7-6113F059F929}") = "Azure.ResourceManager.SpringAppDiscovery.Tests", "tests\Azure.ResourceManager.SpringAppDiscovery.Tests.csproj", "{48CAB45E-35D1-4F37-8DAE-9146BBC083CA}"
+EndProject
+Project("{D5740887-B929-4A8E-A2B7-6113F059F929}") = "Azure.ResourceManager.SpringAppDiscovery.Samples", "samples\Azure.ResourceManager.SpringAppDiscovery.Samples.csproj", "{55EE66FE-62A1-451D-BA43-C10FE3382F66}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "sdkTest", "sdkTest\sdkTest.csproj", "{8D1F559A-21E0-4079-B9EC-98823F799151}"
+EndProject
```

Comment:
what is this project? maybe we should remove it because it does not exist.

Reply:
Removed it